packages/grid/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -317,6 +318,11 @@ export interface DataGridPropsWithDefaultValues {
    * @default false
    */
   disableColumnResize: boolean;
+  /**
+   * If `true`, reordering rows is disabled.
+   * @default false
+   */
+  disableRowReorder: boolean;
```

Comment:
You should be able to define this one in `DataGridProPros` and don't put a default value on `DataGrid` because the hook `useGridRowReorder` is not called at all on the community plan.

See `disableColumnPinning` for equivalent

Reply:
Yes, I tried that but the problem is that I need to access `rootProps.disableRowReorder` inside the `GridRow.tsx` because otherwise even tho the feature hook doesn't run the drag events and the `draggable` are still added to the demo and you are able to visually drag a row.

This feature has more in common with the `disableColumnReorder` in terms of the setup.

Reply:
True if it is used in a community component we are stuck :/